src/pocketmine/level/Position.php

```diff
@@ -89,6 +89,62 @@ public function setLevel(Level $level = null){
 		return $this;
 	}
 
+	/**
+	 * @param Position|int $x
+	 * @param int         $y
+	 * @param int         $z
+	 *
+	 * @return Vector3
+	 */
+	public function add($x, $y = 0, $z = 0) : Position{
+		if($x instanceof Position){
+			return new Position($this->x + $x->x, $this->y + $x->y, $this->z + $x->z);
+		}else{
+			return new Position($this->x + $x, $this->y + $y, $this->z + $z);
+		}
+	}
+	
+	/**
+	 * @param Position|int $x
+	 * @param int         $y
+	 * @param int         $z
+	 *
+	 * @return Vector3
+	 */
+	public function subtract($x = 0, $y = 0, $z = 0) : Position{
+		if($x instanceof Position){
+			return $this->add(-$x->x, -$x->y, -$x->z);
+		}else{
+			return $this->add(-$x, -$y, -$z);
+		}
+	}
+	
+	public function multiply(float $number) : Position{
```

Comment:
I don't see the point of multiplying a `Position`. A `Position`, by definition, is a positional vector. There is no point of scalar-multiplying a positional vector.